inc/TrkInfo.hh

```diff
@@ -15,13 +15,22 @@ namespace mu2e
 {
 // information about the track fit at a particular place
   struct TrkFitInfo {
-    Float_t _fitmom,_fitmomerr;
-    helixpar _fitpar;
-    helixpar _fitparerr;
+    Float_t _mom, _momerr; // momentum and its uncertinaty [MeV/c]
+    Float_t _time; // time of helix at this segment [ns]
+    Float_t _phi; // azimuthal angle of helix at this segment [rad]
+
+    // LoopHelix parameters
+    Float_t _rad, _raderr; // radius of helix [mm]
```

Comment:
I would prefer to use  an array or some similar structure for these parameters, to allow compatibility with different parameterizations supported by KinKal.  Or we could template this struct on KTRAJ as is done inside KinKal, and get the leaf names from that.